base/ca/src/main/java/com/netscape/ca/CertificateAuthority.java

```diff
@@ -1550,34 +1550,35 @@ public OCSPResponse validate(OCSPRequest request)
          *    CertID in the request.
          *
          * 2. If this CA is *not* the issuer, look up the issuer
-         *    by its DN in CAEngine.  If not found, fail.  If
-         *    found, dispatch to its 'validate' method.  Otherwise
-         *    continue.
+         *    by its DN in CAEngine. If found, dispatch to its 'validate'
+         *    method. Otherwise continue.
          *
          * 3. If this CA is NOT the issuing CA, we locate the
          *    issuing CA and dispatch to its 'validate' method.
          *    Otherwise, we move forward to generate and sign the
          *    aggregate OCSP response.
          */
```

Comment:
@fmarco76 I suggest that you request @frasertweedale for a review here since it seems have touched the lightweight CA area.

Reply:
On second thought, looking closely, although it added additional name/key hash checks, the code path should be the same for the lightweight CA case.  I think it should be safe for v10.13 for now.  We can hold off porting to other branches till we hear back from @frasertweedale

Reply:
Yes, the path is the same. The reason to look at the right CertificationAuthority is to get the correct DN and Public Key to verify. The improvement could be to perform the test for each certificate in the request.